src/libsync/clientsideencryption.cpp

```diff
@@ -1248,6 +1248,12 @@ void ClientSideEncryption::encryptPrivateKey(const AccountPtr &account)
 }
 
 void ClientSideEncryption::decryptPrivateKey(const AccountPtr &account, const QByteArray &key) {
+    if (!account->askUserForMnemonic()) {
+        qCInfo(lcCse) << "Not allowed to ask user for mnemonic";
```

Comment:
I would go with debug level otherwise we spam with too many log output in Info level

Reply:
Changed to a `qCDebug`